Packages/eu.netherlands3d.indicators/Runtime/Scripts/UI/HorizontalBarGraph.cs

```diff
@@ -56,6 +60,15 @@ public void SetValue(float value , bool updateFill = true)
             }
         }
 
+        public void SetInformationButtonURL(string url)
+        {
+            if(string.IsNullOrEmpty(url))
```

Comment:
Zou het beter zijn om actief de information button's SetActive hier op false te zetten? Voor wanneer deze methode wordt aangeroepen met een lege waarde of empty? (Public methode, dus zou kunnen)

Reply:
Oh, en dus inclusief dat de listener eraf gaat.